src/Foundatio/Caching/CacheClientHelpers.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace Foundatio.Caching
```

Comment:
Yeah, I think it would be good to have it right next to the interface (it's what we've been doing with all the other abstractions)

https://github.com/exceptionless/Foundatio/blob/master/src/Foundatio/Queues/IQueue.cs#L48